### DISCUSSION TITLE
Minimum lenght for username of 2?

### DISCUSSION THREAD

**Discussion by psychowood** [henrygd/beszel#26] [General]
Not really a bother (hence the discussion and not an issue :) ), but I'm trying to understand if I'm missing something that drives the limit.

Is there a technical limitation for userid length? I have authelia set up and most of my users are 2 chars, I usually create the same userid on every system (name+surname initials unless there are collisions) that is easier to handle for non technical users.

**Comment by henrygd** [comment]
Users is a default PocketBase collection with internal validations. I don't think there's an easy way to lower the min length below three.
